core/algorithms/onpolicy_sync/runner.py

```diff
@@ -420,6 +427,9 @@ def metric_path(self, start_time_str: str) -> str:
         )
 
     def save_project_state(self):
+        if self.disable_config_saving:
+            return
+
```

Comment:
Can we instead do something like:
```python
if not self.disable_config_saving:
    self.save_project_state()
```
wherever `save_project_state` is called? I know it's unlikely but I could imagine someone not wanting configs to save during training but perhaps having some manual call to `save_project_state` in some script.

Reply:
Sure, I'm changing it now 👍